.gitlab-ci.yml

```
@@ -959,25 +959,25 @@ build_img_docker_el8_release:
     ACTIVE_BUILDS: 'pfbuild-centos-8'
 
 # Debian
-build_img_docker_deb11_devel:
+build_img_docker_deb12_devel:
```

```suggestion
build_img_docker_deb12_devel:
```
I am wondering if we should have just name it "deb" all gitlab instead of "deb12".
Then it will follow the same idea as "PFBUILD_DEB_IMG"

It will make sens for the next OS upgrade.
It is just a thinking, I did not check all implications.